setup.py

```diff
@@ -69,7 +70,10 @@ def run(self):
                 "--message-format=json",
             ]
 
-            if platform.machine() in ("ppc64le", "ppc64", "powerpc"):
+            if (
+                platform.machine() in ("ppc64le", "ppc64", "powerpc")
+                or get_platform() == "win-arm64"
```

Comment:
Can we use `platform.platform()` instead of setuptools's private API?

Reply:
I've used`distutils/get_platform` to be able to do cross-compilation if required in the future. IIRC platform module would return host machines configuration so if I try to build arm64 binary wheel from x64 windows then it wouldn't work.

Also `platform.platform()` returns `Windows-10-10.0.19041-SP0` on my surface pro which is probably not what we want.

I think either we can use platform.machine() and sys.platform() to check on win/arm64 or use get_platform from distutils.

Reply:
> I think either we can use platform.machine() and sys.platform() to check on win/arm64

This is fine. I just don't want it use private/deprecated APIs.

Reply:
Yes replaced get_platform with platform.machine() and sys.platform